typhos/utils.py

```diff
@@ -1651,3 +1651,56 @@ def raise_window(widget):
     window.raise_()
     window.activateWindow()
     window.setFocus()
+
+
+class FrameOnEditFilter(QtCore.QObject):
```

Comment:
What's the breakpoint on moving something like this to a utility module vs vendoring it everywhere it is needed? It's pretty short and might want to be customized a bit so I'm not sure

Reply:
I did have to tweak it a bit actually, though maybe we should be putting things into `pcdswidgets` more often?  

(sidebar, using the `finishedEditing` signal requires the FocusIn/FocusOut events to pass through, which mucks around with some of the formatting)